pkg/controllers/execution/execution_controller.go

```diff
@@ -253,8 +261,31 @@ func (c *Controller) tryCreateOrUpdateWorkload(ctx context.Context, clusterName
 	return nil
 }
 
+func (c *Controller) updateWorkDispatchingConditionIfNeeded(ctx context.Context, work *workv1alpha1.Work) error {
```

Comment:
Just a question: In addition to setting `Condition`, should we also send some `event`s?

Reply:
Even if there is an event, that's the `Normal` type(Not a warning) I don't know if this event is desired.
I guess @XiShanYongYe-Chang can help track this on the umbrella issue, we can add it by another PR if there is a need.

Reply:
@chaunceyjiang has mentioned adding an event at the community meeting, it's something, we can do it,  

let me track it in the umbrella issue, and do it in another pr.

Reply:
https://github.com/karmada-io/karmada/pull/5332